src/python/pants/backend/python/util_rules/pex_environment.py

```diff
@@ -29,6 +30,23 @@ class PexRuntimeEnvironment(Subsystem):
     options_scope = "pex"
     help = "How Pants uses Pex to run Python subprocesses."
```

Comment:
The new option should live in `[python]`, not `[pex]`, per this `help` message. 

Maybe `[pex].run_pex_binaries_with_deprecated_semantics`, with default to `True`. Then in 2.14 we change default to `False`, and in 2.15 we remove the option.

Reply:
Maybe we could change the help? To me, `pex_binary` behavior goes in `pex`. Then again `pex-binary-defaults` could also live in `pex`. Maybe we should come up with a "subsystem conventions" guide :thinking: 

Also `run_pex_binaries_with_deprecated_semantics` doesn't tell the user what the option is controlling. I'd rather call a spade a spade here. Not to mention the current semantics itself is not intuitive.

I'm open to suggestions, but I want to make sure they "fit".

Reply:
I still don't think this is right. This new option does not fit at all with the other options.